src/components/item/settings/ItemSettings.tsx

```diff
@@ -203,6 +211,12 @@ const ItemSettings = ({ item }: Props): JSX.Element => {
 
   return (
     <Container disableGutters sx={{ mt: 2 }}>
+      <Alert severity="warning" sx={{ marginBottom: 2 }}>
+        {translateBuilder(BUILDER.UPDATE_THUMBNAIL_AT_INFO_ALERT)}{' '}
+        <Link to={buildItemInformationPath(item.id)}>
+          {translateBuilder(BUILDER.UPDATE_THUMBNAIL_LINK_TEXT)}
```

Comment:
I think it's possible to use only one component here using this https://react.i18next.com/latest/trans-component